hyperliquid/exchange.py

```diff
@@ -421,12 +519,12 @@ def usd_class_transfer(self, amount: float, to_perp: bool) -> Any:
 
     # Deprecated in favor of usd_class_transfer
     def user_spot_transfer(self, usdc: float, to_perp: bool) -> Any:
-        usdc = int(round(usdc, 2) * 1e6)
+        usdc_rounded_micros = int(round(usdc, 2) * 1e6)
```

Comment:
I'd prefer not making this rename. usdc has 6 decimals so this is the actual amount of usdc and is not rounded. Perhaps the input should have been usd instead of usdc, but renaming it now would be a breaking change